<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="18634" started-at="2019-10-09T03:22:26Z" finished-at="2019-10-09T03:22:45Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="18634" started-at="2019-10-09T03:22:26Z" finished-at="2019-10-09T03:22:45Z">
      <class name="testcases.LoginTest">
        <test-method status="PASS" signature="setupReport()[pri:0, instance:testcases.LoginTest@8bd1b6a]" name="setupReport" is-config="true" duration-ms="280" started-at="2019-10-09T11:22:26Z" finished-at="2019-10-09T11:22:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupReport -->
        <test-method status="PASS" signature="setupBrowsers()[pri:0, instance:testcases.LoginTest@8bd1b6a]" name="setupBrowsers" is-config="true" duration-ms="9076" started-at="2019-10-09T11:22:26Z" finished-at="2019-10-09T11:22:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupBrowsers -->
        <test-method status="FAIL" signature="loginToHRM()[pri:1, instance:testcases.LoginTest@8bd1b6a]" name="loginToHRM" duration-ms="7632" started-at="2019-10-09T11:22:36Z" finished-at="2019-10-09T11:22:43Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at dataProviderFactory.ExcelDataProvider.getCellData(ExcelDataProvider.java:30)
	at pages.AddUserHRM.employPassword(AddUserHRM.java:92)
	at testcases.LoginTest.loginToHRM(LoginTest.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToHRM -->
        <test-method status="PASS" signature="appendReport(org.testng.ITestResult)[pri:0, instance:testcases.LoginTest@8bd1b6a]" name="appendReport" is-config="true" duration-ms="932" started-at="2019-10-09T11:22:43Z" finished-at="2019-10-09T11:22:44Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginToHRM status=FAILURE method=LoginTest.loginToHRM()[pri:1, instance:testcases.LoginTest@8bd1b6a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- appendReport -->
        <test-method status="PASS" signature="closeSessions()[pri:0, instance:testcases.LoginTest@8bd1b6a]" name="closeSessions" is-config="true" duration-ms="962" started-at="2019-10-09T11:22:44Z" finished-at="2019-10-09T11:22:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeSessions -->
      </class> <!-- testcases.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
